frontend/exporter/src/types/ty.rs

```diff
@@ -796,6 +796,16 @@ pub enum TyKind {
     Uint(UintTy),
     Float(FloatTy),
 
+    #[custom_arm(
+        ty::TyKind::FnDef(def_id, generics) => {
+            let tcx = s.base().tcx;
+            let sig = tcx.fn_sig(*def_id).instantiate(tcx, generics);
+            TyKind::FnDef(def_id.sinto(s), Box::new(sig.sinto(s)))
+        },
+    )]
+    /// Reflects [`ty::TyKind::FnDef`]
+    FnDef(DefId, Box<PolyFnSig>),
```

Comment:
Can you store the generics as well, we'll need them. This will require both a `Vec<GenericArg>` and a `Vec<ImplExpr>` obtained via `solve_item_required_traits`